frontend/scripts/openapi.yaml

```diff
@@ -1973,6 +1973,7 @@ components:
       - tournesol_score
       - type
       - uid
+      - metadata
```

Comment:
I added that as required field because otherwise an `EntityCard` doesn't accept an `Entity` object because `metadata` may be undefined. But I'm not sure the backend always includes this attributes. If it doesn't we can also change `EntityCard` to make it handle an undefined `metadata`.

Reply:
This file is generated by `yarn update-schema` so if you've manually added `metadata` I'm scared it will be automatically removed the next time the command is ran.

I'll probably be working on something else until the end of the week, so I can't suggest a proper solution for now, but let's keep in mind this should be fixed before merging.

Reply:
If we change the API it won't be necessary anymore because the new API can return an `EntityNoExtraField`. But it's probably cleaner anyway to make the common attributes between `EntityNoExtraField` and `Entity` have the same requiredness.